test/Microsoft.Azure.WebJobs.Host.FunctionalTests/FunctionalTest.cs

```diff
@@ -213,73 +213,28 @@ internal static class FunctionalTest
                 StorageAccount = storageAccount
             };
             IHostIdProvider hostIdProvider = new FakeHostIdProvider();
-            INameResolver nameResolver = null;
```

Comment:
also very happy to remove this.  Large section of very tricky test code that duplicated product code with subtle changes ... good to remove it and just use the product code.